MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
```

Comment:
[Optional] I recommend you replace [YOUR NAME] in the license file with your actual name and [APP TYPE] with a matching app type e.g website. This will customize the file to match your usage.

Reply:
Thank you @emmiiorji, I have done all the recommendations 🙃